src/global.css

```
@@ -0,0 +1,22 @@
+@import 'tailwindcss';
+@import 'uniwind';
+
+@layer theme {
+  :root {
+    @variant light {
```

Stylelint (17.3.0)

[error] Unexpected unknown at-rule "@variant" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)